app/Base/Controllers/DataTableController.php

```diff
@@ -162,7 +174,7 @@ protected function getColumns()
         $columns = [];
         $model = $this->getModelName();
         $table = $this->getTableName();
-        foreach (array_merge($this->image_columns, $this->columns, $this->boolean_columns) as $column) {
+        foreach (array_merge($this->image_columns, $this->columns, $this->boolean_columns, $this->count_columns) as $column) {
```

Comment:
I think that, we can also add a control logic for deciding whether they can be queried or not, assuming that the count columns are joined at the end of the merge operation. 

```php
$array = array_merge($this->image_columns, $this->columns, $this->boolean_columns, $this->count_columns);
foreach ($array as $key => $column) {
    $string = join([$table, $column], ".");
    $title = trans('admin.fields.' . join([$model, $column], '.'));
    $orderAndSearch = $key < (count($array) - count($this->count_columns)) ? true : false;
    array_push($columns, [
        'data' => $column, 'name' => $string, 'title' => $title,
        'orderable' => $orderAndSearch, 'searchable' => $orderAndSearch
     ]);
}
```